### DISCUSSION TITLE
Show both of UTF-8 and SJIS in Encoding box

### DISCUSSION THREAD

**Discussion by shirayuki** [gitextensions/gitextensions#10034] [Q&A]
### Feature description

Always show the following encodings in Encodings box in Japanese environment:
- Unicode (UTF-8)
- Japanese (Shift JIS)

Or provide a way to customize Encodings list.

![image](https://user-images.githubusercontent.com/18039/172721393-d6afb665-3ea5-4bfe-818a-492e68f7d4f9.png)
![image](https://user-images.githubusercontent.com/18039/172723136-45f21e9c-685a-4a9b-ac3b-916a1bfcd805.png)

### Environment

- GitExtensions version:
- GIT version: 
- OS version: Windows 10 Professional
- .NET version:

**Comment by RussKie** [comment]
![image](https://user-images.githubusercontent.com/4403806/173184913-954ac282-9960-4458-ab2e-04e1d5ec4bbb.png)
